src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -275,6 +275,23 @@ public async Task ConfigureAsync(CommandSettings command)
                 throw new NotSupportedException("Message queue listen OAuth token.");
             }
 
+            // Configure to use fips-compliant encryption schemes for communication if required by server
```

Comment:
Moved up so that the crypto settings are available before doing the connection test